### ISSUE TITLE
 I donot understand what is the meaning of ‘’maximize D(x)-1-D(G(z))‘’ ?

### ISSUE THREAD

**Issue by czjghost** [leftthomas/SRGAN#105]
It's my first time to touch with GAN, and I try to search the original paper. I have understood the loss function of Generator net writed in your code(maybe it refers to ESRGAN‘s and adds TV loss to denoise), but I found that the loss function of Discriminator net is different from the paper of SRGAN  and ESRGAN(or maybe I make mistake understanding it).....could you help me explain the loss function of Discriminator net writed in your code or give me some advise to solve it? thanks very much!!! （if I change the code "d_loss = 1 - real_out + fake_out" to code “d_loss = - log(real_out ) - log(1 - fake_out)”, will it work? I found your loss function will get Score_D and Score_G close to zero after training of some epoch)

**Comment by czjghost** [comment]
I found "https://github.com/amanchadha/iSeeBetter" have same code of your project...I want to run it finally, but I want to understand SRGAN and RBPN respectively
